File: dofun/safe/deviceinfo/info/NetWorkInfo.java
package com.dofun.safe.deviceinfo.info;

import android.content.Context;
import androidx.core.util.Pair;
import com.dofun.safe.deviceinfo.utils.CommandUtils;
import com.dofun.safe.deviceinfo.utils.NetWorkUtils;
import java.util.ArrayList;
import java.util.List;
import org.json.JSONObject;

/* loaded from: classes8.dex */
public class NetWorkInfo {
    public static JSONObject getNetWorkInfoJson(Context context) {
        return CommandUtils.convertToJson(getNetWorkInfo(context));
    }

    public static List<Pair<String, String>> getNetWorkInfo(Context context) {
        List<Pair<String, String>> list = new ArrayList<>();
        getNetWorkStatus(context, list);
        return list;
    }

    private static void getNetWorkStatus(Context context, List<Pair<String, String>> list) {
        list.add(new Pair<>("netAvailability", NetWorkUtils.isNetworkConnected(context) + ""));
        list.add(new Pair<>("mobileAvailability", NetWorkUtils.isMobileEnabled(context) + ""));
        list.add(new Pair<>("wifiAvailability", NetWorkUtils.isWifi(context) + ""));
        list.add(new Pair<>("netType", NetWorkUtils.getNetWorkType(context)));
    }
}

File: app/src/main/java/com/dofun/safe/deviceinfo/info/NetWorkInfo.kt
package com.dofun.safe.deviceinfo.info

import android.content.Context
import androidx.core.util.Pair
import com.dofun.safe.deviceinfo.utils.CommandUtils
import com.dofun.safe.deviceinfo.utils.NetWorkUtils
import org.json.JSONObject

object NetWorkInfo {
    @JvmStatic
    fun getNetWorkInfoJson(context: Context): JSONObject {
        return CommandUtils.convertToJson(getNetWorkInfo(context))
    }

    @JvmStatic
    fun getNetWorkInfo(context: Context): List<Pair<String, String>> {
        val list = ArrayList<Pair<String, String>>()
        getNetWorkStatus(context, list)
        return list
    }

    private fun getNetWorkStatus(context: Context, list: MutableList<Pair<String, String>>) {
        list.add(Pair("netAvailability", NetWorkUtils.isNetworkConnected(context).toString()))
        list.add(Pair("mobileAvailability", NetWorkUtils.isMobileEnabled(context).toString()))
        list.add(Pair("wifiAvailability", NetWorkUtils.isWifi(context).toString()))
        list.add(Pair("netType", NetWorkUtils.getNetWorkType(context)))
    }
}
